.github/workflows/ci.yml

```diff
@@ -72,7 +72,6 @@ jobs:
         # When writing back to the repo, we write to directories based
         # on the bindplane version.
         bindplane_versions:
-          - 1.45.0 # keep
```

Comment:
1.45 is not particularly interesting and did not support some of the new test resources I added.